locksmith/src/app.ts

```diff
@@ -35,11 +35,7 @@ app.use(cookieParser())
 // Request logging
 app.use(
   expressWinston.logger({
-    transports: [
-      new winston.transports.Console({
-        silent: process.env?.NODE_ENV === 'test',
-      }),
-    ],
+    transports: [new winston.transports.Console({})],
```

Comment:
We kind of want to silence in tests though?

Reply:
They are helpful in figuring out why a test isn't working in the CI hence removed.

Reply:
ok